src/domain/interfaces/account.datasource.interface.ts

```diff
@@ -96,7 +104,7 @@ export interface IAccountDataSource {
     chainId: string;
     safeAddress: string;
     signer: string;
-  }): Promise<void>;
+  }): Promise<Account>;
```

Comment:
Are these return types changes related to this PR? 

The return type for this one in particular is not used anywhere AFAIK.

Reply:
Not exactly related no. It came from an iteration of returning always the updated account. I will remove it as we don't use the resulting type currently.

Reply:
Removed in [1029720](https://github.com/safe-global/safe-client-gateway/pull/1101/commits/1029720c35fe0377c02abd0e9d0abe197d2ae609)